.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm-debug.log
 lib/api.json
+.vscode/*
```

Comment:
Didn't even know that existed :o !
Thanks for the heads up on that 

Let me know if it's all good :D !